featurebyte/worker/task/batch_feature_create.py

```diff
@@ -149,5 +160,12 @@ async def execute(self) -> Any:
                 raise DocumentInconsistencyError("Inconsistent feature definition detected!")
 
             # update the progress
-            percent = 100 * (i + 1) / total
+            percent = start_percentage + (end_percentage - start_percentage) * (i + 1) / total
```

Comment:
Another way is to pass in a ranged progress callback:

https://github.com/featurebyte/featurebyte/blob/a1f7dd966783d52f30fdebddfe9d60c67dc21649/featurebyte/common/progress.py#L9-L16

so this kind of arithmetic can be performed automatically.

Reply:
Will update this to use range process callback. 👍